README.md

```diff
@@ -180,6 +180,10 @@ rules on top of those in the configuration file.
 fortitude check --extend-select=M
 ```
 
+## Documentation
+
+See [table of rules](docs/all_rules.md) for a list of all rules.
```

Comment:
```suggestion
See [table of rules](docs/rules.md) for a list of all rules.
```

Reply:
Pointed this to readthedocs instead (although link won't work until this is merged)